src/types/index.tsx

```diff
@@ -50,3 +50,17 @@ export type Session = {
   start_date: string;
   year: number;
 };
+
+export type Attendance = {
+  date: string;
+  attendees: number[];
+};
+
+export type ClassInfo = {
```

Comment:
does ts absolutely hate us if we use `Class` :')

Reply:
WOW it actually works with Class let's gooo

thanks for catching that

Reply:
idk why I just assumed it would break 😆